lib/app/app.rb

```diff
@@ -30,7 +30,7 @@ class ConnectorsWebApp < Sinatra::Base
     set :raise_errors, false
     set :show_exceptions, false
     set :bind, settings.http['host']
-    set :port, settings.http['port']
+    set :port, [ENV['PORT'], settings.http['port'], '9292'].detect(&:present?)
```

Comment:
I like this. No reason to do it in this PR, but we should add these env var overrides for all of these, which will make docker config a lot nicer.